src/version.h

```diff
@@ -32,4 +32,16 @@
 #define PLAYER_ADDTL ""
 #define PLAYER_VERSION TO_STRING(PLAYER_MAJOR) "." TO_STRING(PLAYER_MINOR) "." TO_STRING(PLAYER_PATCH)
 
+/**
+ * Version written to the easyrpg_data.version field in savegames.
+ * Update this number when savegame changes cause incompatibility with old
+ * savegames.
+ *
+ * 1st digit (if not 0): Major
+ * 2nd digit: Minor
+ * 3rd digit: Patch
+ * 4rd digit: Usually 0. Increment only when having a good reason.
+ */
+#define PLAYER_SAVEGAME_VERSION 600
```

Comment:
This should just be something like `PLAYER_MAJOR * 1000 + PLAYER_MINOR * 100 + PLAYER_PATCH * 10`

Then we don't need to update it separately every time and your comments become unnecessary.

Reply:
but we will not change the savegame format with every release.

Reply:
We think we won't change it. That doesn't mean a save game bug can't show up in one particular release inadvertently. If we don mark every release, we wouldn't be able to tag this after the fact.

Reply:
hm actually this could be also interesting for statistics, because then we can see in the logs which player version was used before for creating the save.
I will think about a solution.

Reply:
Yeah, that would be a great boon to debugging save game bug reports.